rest/route/admin_restart.go

```diff
@@ -21,11 +21,12 @@ func makeRestartRoute(sc data.Connector, queue amboy.Queue) gimlet.RouteHandler
 }
 
 type restartHandler struct {
-	StartTime  time.Time `json:"start_time"`
-	EndTime    time.Time `json:"end_time"`
-	DryRun     bool      `json:"dry_run"`
-	OnlyRed    bool      `json:"only_red"`
-	OnlyPurple bool      `json:"only_purple"`
+	StartTime          time.Time `json:"start_time"`
+	EndTime            time.Time `json:"end_time"`
+	DryRun             bool      `json:"dry_run"`
+	IncludeTestFailed  bool      `json:"inc_test_failed"`
```

Comment:
Nit but you might as well spell out "include". I read this and though "increment? what?"

Reply:
done